include/tester/db/Database.hpp

```diff
@@ -169,17 +95,15 @@ class Database
 
 
 
-    bsoncxx::stdx::optional<bsoncxx::document::value> getLog(const std::string& collection_name, const std::string& id)
+    bsoncxx::stdx::optional<bsoncxx::document::value> getLog(std::string collectionName, std::string id)
     {
-        mongocxx::collection collection = _database[collection_name];
-        bsoncxx::stdx::optional<bsoncxx::document::value> result = collection.find_one(bsoncxx::builder::stream::document{} 
+        mongocxx::collection collection = _database[collectionName];
+        return collection.find_one(bsoncxx::builder::stream::document{} 
             << "_id" << bsoncxx::oid{id} << bsoncxx::builder::stream::finalize);
-
-        return result;
     }
 
 
-    bsoncxx::stdx::optional<bsoncxx::document::value> get(const std::string &collaction_name,  const std::string &id, const std::string &time)
+    bsoncxx::stdx::optional<bsoncxx::document::value> get(std::string collaction_name,  std::string id, std::string time)
```

Comment:
Look carefully here - if you're passing `id` and `time` by value - make sure of the following:
- if you're using them only once - use `std::move` on that single occurance.
- if you're using them multiple times - pass them by `const &` instead.

Reply:
I consider to delete this function I never use it, because I use only `getLog` function. What do you think?

Reply:
if `get` is `VoIP-Tester` specific and you're not using it - remove it.
if it is a generic method - keep it.

Reply:
done